### ISSUE TITLE
[RTL] Navigation arrows are reversed

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6538]
Actual results:
The navigation buttons on bottom right are reversed.

Expected results:
The should be turning away from each other.

Correct:
![Screen Shot 2020-05-06 at 1 59 16 PM](https://user-images.githubusercontent.com/32130829/81169997-86f68c00-8fa2-11ea-981b-7806f186de7d.png)

Incorrect: 
![Screen Shot 2020-05-06 at 1 59 09 PM](https://user-images.githubusercontent.com/32130829/81169985-7e9e5100-8fa2-11ea-8a67-7657833ad36f.png)

Via: https://bugzilla.mozilla.org/show_bug.cgi?id=1483556

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2393)

**Comment by dnarcese** [comment]
The screenshot has the old icons so that means it isn't a new regression due to the icon change. This has probably been around for awhile.

**Comment by SimonBasca** [comment]
Verified fix on v98.0 (8023).
